atom_batch_execution/scripts/batch_execution

```diff
@@ -170,6 +198,15 @@ def main():
         # Start executing command.
         execute(experiment['cmd'], verbose=args['verbose'], save_path=experiment_folder)
 
+        # Save experiment settings
+        for experiment_type in data['experiments']:
+            if experiment_type["name"] == stripAutomaticSuffixes(experiment_key,args):
```

Comment:
Is this a standardize way to save experiments? This should be documentated somewhere. Please correct me if it is.

Reply:
This hunk of code I added its not "saving the experiment". Its saving the settings used for a specific experiment.

I'm going to give a example from a random experiment I had saved from my MSc.

In the `data.yml` of this experiment I have one experiment with these settings : 

```yaml
 - {name: nig_0.15-no_odom_calib, nig_value: 0.15, ntfv_value: 0.0,  calibrate_odom : false}
```

So the script saves in the experiment folder (where all the experiment files were already being collected) a file named `nig_0.15-no_odom_calib_settings.yml` which contains the following : 

```yaml
   name: nig_0.15-no_odom_calib
   nig_value: 0.15
   ntfv_value: 0.0
   calibrate_odom: false
```

This file is then used in the plotting script I developed to filter experiments according to a specified criteria using a lambda function. This was necessary because the name of each folder in the results folder could not fully decode the settings of a experiment and each file within couldn't neither.